src/coreclr/jit/compiler.h

```diff
@@ -9726,6 +9726,17 @@ class Compiler
         return compOpportunisticallyDependsOn(InstructionSet_APX);
     }
 
+    //------------------------------------------------------------------------
+    // canUseApxEvexEncoding - Answer the question: Are APX-EVEX encodings supported on this target.
+    //
+    // Returns:
+    //    `true` if APX-EVEX encoding is supported, `false` if not.
+    //
+    bool canUseApxEvexEncoding() const
+    {
+        return canUseApxEncoding() && canUseEvexEncoding();
```

Comment:
Can you remind me under what scenario you can have APX but not EVEX?

I had thought that if REX2 was available then EVEX was as well. So is this trying to support a specific scenario or can we just check for `APX`?